<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectFileNewDialog</class>
 <widget class="QDialog" name="ProjectFileNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component type choice</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <item>
    <widget class="QLabel" name="labelComponentType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>What do you want to do?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editName">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbNewComponent">
     <property name="text">
      <string>Create new component</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbExistingComponent">
     <property name="text">
      <string>Add existing component</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbImportFromProject">
     <property name="text">
      <string>Import component from existing project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>ProjectFileNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>ProjectFileNewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
